### ISSUE TITLE
Error on Unblock-file

### ISSUE THREAD

**Issue by Sigri44** [MultiPoolMiner/MultiPoolMiner#1013]
Hi,

I have error when i lauch the start.bat, yesterday all works, but today, i have this error. (see the attachment)
![start](https://user-images.githubusercontent.com/9304947/34865986-238aa2f8-f77b-11e7-85c3-7e2dd2757043.png)

The file is unblock (right clic, proprieties), and no AV analyse this folder..

I have lauch start.bat with Administrator right or not, there is the same error.

Thx

**Comment by aaronsace** [comment]
Why have covered up the file location???

**Comment by Sigri44** [comment]
But it's confidential ! The script is on the Download folder of user ;)

**Comment by aaronsace** [comment]
Probably need to review your security if you have confidential information in folder locations. 
Anyway, the location can't be the issue if it's just the default downloads folder. 

Why are you running the script as admin? What happens in normal user mode i.e. just double click on the included 'start.bat' file?

**Comment by Sigri44** [comment]
Not that I do not want my tree to be visible on the internet;)

If I run the start.bat I have the same error.

But the script does not stop by itself, it launches the profitability but no miner does this run ..!

**Comment by aaronsace** [comment]
Please upload log files.

**Comment by Sigri44** [comment]
https://pastebin.com/TKzgVb0Y

**Comment by Sigri44** [comment]
https://github.com/MultiPoolMiner/MultiPoolMiner/issues/1013#issuecomment-357309725

And This WTF profit last month x)

**Comment by Sigri44** [comment]
When the script begin, i have "Status : Failed", and it say excavator launch 3 times but 0Hs. It's normal but no any windows is open, juste start.bat, it refresh but no mining launch.. :/

**Comment by aaronsace** [comment]
@Sigri44 No idea as to what issue you want me to look at anymore. 

Upload the log file(s) to the GitHub post if you want me to look at something.

**Comment by Sigri44** [comment]
The pastebin link is the full of log today (U don't know pastebin.. ?)

**Comment by aaronsace** [comment]
@Sigri44 Nothing on it.

**Comment by aaronsace** [comment]
Where is all this profit and excavator stuff?

**Comment by aaronsace** [comment]
Is the unblock issue resolved as well; it's not in there either...?

**Comment by aaronsace** [comment]
Please use GitHub not external sites.

**Comment by Sigri44** [comment]
**********************
PowerShell transcript start
Start time: 20180112092742
Username: xxxxxxx\xxxxxx
RunAs User: xxxxxxx\xxxxxx
Configuration Name: 
Machine: xxxxxxx(Microsoft Windows NT 6.1.7601 Service Pack 1)
Host Application: C:\Program Files\PowerShell\6.0.0-rc.2\pwsh.dll -noexit -executionpolicy bypass -windowstyle maximized -command & .\multipoolminer.ps1 -wallet 34Dn2sYFXXXXXXXXXXXXXXXXXXiD1W5 -username xxxxxx -workername multipoolminer -region europe -currency btc,usd,eur -type nvidia,cpu -poolname miningpoolhub,miningpoolhubcoins,zpool,nicehash -algorithm cryptonight,decred,decrednicehash,ethash,ethash2gb,equihash,groestl,lbry,lyra2re2,lyra2z,neoscrypt,pascal,sia,siaclaymore,sianicehash,sib,skunk -donate 1 -watchdog -switchingprevention 2 -MPHApiKey 3059e9ebbf626e75c3axxxxxxxxxxxxxxxxxxxxxxxxxxx2666a2a376
Process ID: 4312
PSVersion: 6.0.0-rc.2
PSEdition: Core
GitCommitId: v6.0.0-rc.2
OS: Microsoft Windows 6.1.7601 S
Platform: Win32NT
PSCompatibleVersions: 1.0, 2.0, 3.0, 4.0, 5.0, 5.1.10032.0, 6.0.0
PSRemotingProtocolVersion: 2.3
SerializationVersion: 1.1.0.1
WSManStackVersion: 3.0
**********************
Transcript started, output file is .\Logs\2018-01-12_09-27-42.txt
PS>TerminatingError(): "The pipeline has been stopped."
>> TerminatingError(): "The pipeline has been stopped."
>> TerminatingError(): "The pipeline has been stopped."
>> TerminatingError(): "The pipeline has been stopped."
>> TerminatingError(): "The pipeline has been stopped."

**Comment by aaronsace** [comment]
Upload all the files inside the folder called 'logs'. Drag & drop.

**Comment by Sigri44** [comment]
[2018-01-12_09-23-40.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627523/2018-01-12_09-23-40.txt)
[2018-01-12_09-27-42.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627524/2018-01-12_09-27-42.txt)
[2018-01-12_19-31-55.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627525/2018-01-12_19-31-55.txt)
[2018-01-12_19-33-53.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627526/2018-01-12_19-33-53.txt)
[2018-01-12_19-39-17.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627527/2018-01-12_19-39-17.txt)
[2018-01-12_08-09-34.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627528/2018-01-12_08-09-34.txt)
[2018-01-12_08-11-30.txt](https://github.com/MultiPoolMiner/MultiPoolMiner/files/1627529/2018-01-12_08-11-30.txt)

**Comment by aaronsace** [comment]
None this makes sense. 

'-MPHApiKey' is not part of MultiPoolMiner. 
MultiPoolMiner is working at 08:11 then not working at 09:23 then is working again at 19:31...

The only log that is longer than 2 minutes is 19:39 but it's working fine. 

You appear to have resolved the unblock-file issue. If you are still having problems then post a new issue.

**Comment by Sigri44** [comment]
I have found this in a website for the API key.... :/
Yes many error.
At 19:31 i have delete ethash
Unblock file is always on the launch script but after script launch and failed. I test to download again all good releases and test it, thx !
